js/pricing.js

```diff
@@ -9,17 +12,40 @@ $(
     function () {
         const $confirmPersonalInformation = $('#confirm-personal-information');
         const $confirmDevelopmentAgreement = $('#confirm-development-agreement');
-        const $disabledButton = $('#order-now-btn');
+        const $orderButton = $('#order-now-btn');
+        const $redirectScreen = $('#redirectScreen');
+        const $loader = $('.redirect-loader', '#redirectScreen');
+        const $errorContainer = $('.redirect-error', '#redirectScreen');
```

Comment:
Why do we have 2 selectors? Do we expect more than 1 element? If so, please consider renaming variables to reflect this.